test/eth-bridge-mv2.test.js

```diff
@@ -0,0 +1,121 @@
+const sinon = require('sinon');
```

Comment:
Regarding file name, should the `mv2` terminology be dropped given that this package can (should?) be extension/manifest agnostic? (I left a similar comment on the Ledger MR)

Reply:
The MV2 terminology for the bridge should be maintained, as the bridge for MV2 and MV3 are unlikely to be the same, since we cannot use `TrezorConnect` directly from within the extension.

What it's true, IMO, is that the MV2 and MV3 bridge implementations should live in the extension/mobile/desktop code and not in the keyring, as they might be using different dependencies to achieve that goal.

The only reason I left the MV2 bridge here is to make the transition easier.

Reply:
@bergarces Understood, but given that there could be a non-extension-related library (i.e., a non-mv-related libary) that might want to the `mv2` bridge - i.e., the TrezorConnect bridge, then would it not be more logical to rename this keyring something like `TrezorNativeBridge` instead of `TrezorBridgeMv2`.

Bear in mind that the `TrezorConnect` library can still be used on `mv3` assuming some changes are made to `metamask-extension` i.e., persisting clients etc. My draft MR for MV3 Hardware wallets currently uses TrezorConnect, but it still gets caught on some edge-cases until this issue is resolved: https://github.com/MetaMask/metamask-extension/issues/17649

E.g., Enkrypt uses `TrezorConnect` (albeit via `@trezor/connect-web` for their `mv3` chrome extension without issues: https://github.com/enkryptcom/enKrypt/search?q=trezor%2Fconnect